src/CacheInterface.php

```diff
@@ -59,7 +59,7 @@ public function getOrSet(
         mixed $key,
         callable $callable,
         DateInterval|int|null $ttl = null,
-        Dependency $dependency = null,
+        Dependency|null $dependency = null,
```

Comment:
Maybe `?Dependency $dependency = null`?

Reply:
I kept it with the same style of the package code, which uses type union, which is explicit.